Python/Product/Analysis/AnalysisNetStandard.csproj

```diff
@@ -4,6 +4,12 @@
     <RootNamespace>Microsoft.PythonTools.Analysis</RootNamespace>
     <AssemblyName>Microsoft.Python.Analysis.Engine</AssemblyName>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;1998</NoWarn>
```

Comment:
Can you add a comment what are those warnings?

Reply:
OK